<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormA7105</class>
 <widget class="QWidget" name="FormA7105">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>4 byte data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPingData">
       <property name="text">
        <string>HELO</string>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPing">
       <property name="text">
        <string>Ping</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Response:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPingResponse">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonReadFlashJedecId">
       <property name="text">
        <string>Flash JedecId</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFlashJedecId">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Addr (Hex)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditReadAddr">
       <property name="text">
        <string>00</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>length</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditReadLen">
       <property name="text">
        <string>8</string>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReadData">
       <property name="text">
        <string>Read Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRead4KB">
       <property name="text">
        <string>Read4KB</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDataRead"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Sector (4KB) Addr (Hex)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSectorEraseAddr">
       <property name="text">
        <string>1000</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSectorErase">
       <property name="text">
        <string>Erase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSectorEraseState">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Addr (Hex)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditWriteAddr">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>data(hex)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditWriteData">
       <property name="text">
        <string>01020304</string>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWriteData">
       <property name="text">
        <string>Write Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWrite4KB">
       <property name="text">
        <string>Write4KB</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pushButtonChipErase">
       <property name="text">
        <string>Chip Erase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonWriteBin">
       <property name="text">
        <string>Write Bin Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
